docs/tutorial_email_generator.md

```diff
@@ -220,6 +217,50 @@ semantic_similarity = langcheck.metrics.semantic_similarity(generated_outputs=ge
 
 langcheck.plot.scatter(semantic_similarity, rougeL)
 ```
-![MetricValue output](_static/email_tutorial_scatter.png)
+![MetricValue output](_static/email_tutorial_scatter_1.png)
+
+Due to the longform nature of these generated emails, we can see that the rougeL values tend to be a bit low, even though the semantic_similarity values are high.
+
+However, one aspect that rougeL would be a good fit to evaluate would be the generated subjects at the beginning of the generated emails. Below, we first check that the generated email does indeed follow the `Subject: <appropriate subject>` that we specified in the prompt, and then compute the rougeL score between the generated subject and the reference subject. We can see `rougeL` is able to identify some meaningful differences in the wording of the subject, even when the semantic meanings are pretty similar.
+
+```python
+import re
+
+generated_emails = [
+    item["generated_email"]
+    for item in [japan_trip, running_late, google_application]
+]
+reference_emails = [
+    item["reference"]
+    for item in [japan_trip, running_late, google_application]
+]
+
+# Check that the generated email contains the string "Subject: "
+contains_subject = langcheck.metrics.contains_any_strings(
+    generated_outputs=generated_emails, strings=["Subject: "])
```

Comment:
We could also use `contains_regex()` with your regex below. But I think `contains_any_strings()` is fine too

Reply:
Changed to `contains_regex()`!